lib/screens/config/cache.ex

```diff
@@ -0,0 +1,176 @@
+defmodule Screens.Config.Cache do
+  @moduledoc """
+  Functions to read data from a cached copy of the screens config.
+  """
+
+  alias Screens.Config
+  alias ScreensConfig.Devops
+
+  use Screens.Cache.Client, table: :screens_config
+
+  @type table_contents :: list(table_entry)
+
+  @type table_entry ::
+          {{:screen, screen_id :: String.t()}, ScreensConfig.Screen.t()}
+          | {:last_deploy_timestamp, DateTime.t()}
+          | {:devops, ScreensConfig.Devops.t()}
+
+  def ok?, do: table_exists?()
```

Comment:
👀 where is this used?

Reply:
nvm found it! [lib/screens_web/controllers/screen_api_controller.ex](https://github.com/mbta/screens/pull/1934/files#diff-b2d4a64e8a99103fa5b243e7934673c27d9a53539e80dbd3be167dfb671565b5)